pipelines/rj_smtr/schedules.py

```diff
@@ -107,3 +107,26 @@
         ),
     ]
 )
+
+every_fortnight = Schedule(
```

Comment:
```suggestion
every_dayofmonth_one_and_sixteen = Schedule(
```

deixar o nome mais explicito